tools/federation/src/main/java/org/eclipse/rdf4j/federated/exception/FedXException.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.federated.exception;
+
+/**
+ * Base class for any FedX Exception.
+ * 
+ * @author Andreas Schwarte
+ *
+ */
+public class FedXException extends RuntimeException {
```

Comment:
Could we make this extend RDF4JException?

Reply:
sure, that makes perfectly sense. will be done in the next commit as part of the PR.